pkg/config/config_template.yaml

```diff
@@ -3613,7 +3619,7 @@ api_key:
 ## @param prometheus_scrape - custom object - optional
 ## This section configures the Autodiscovery based on the Prometheus annotations
 #
-# prometheus:
+# prometheus_scrape:
```

Comment:
This section was named differently in the code: https://github.com/DataDog/datadog-agent/blob/ba08dfcc0bb4b569fc5a59e65652b46eeeba42a4/pkg/config/setup/config.go#L395-L398